src/MOLECULE/bond_fene.cpp

```diff
@@ -95,7 +89,7 @@ void BondFENE::compute(int eflag, int vflag)
 
     // force from LJ term
 
-    if (rsq < TWO_1_3*sigma[type]*sigma[type]) {
+    if (rsq < MY_CBRT2*sigma[type]*sigma[type]) {
```

Comment:
Please change this to MY_CUBEROOT2 - it went from being obvious (at least to me) to no idea what it is